.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,29 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+#        name: Checkout repo
+#        with:
+#          ref: ${{ github.event.pull_request.head.ref }}
+#          fetch-depth: 0
+      - uses: docker/build-push-action@v1.1.0
+        with:
+          name: Build and push image
+          registry: registry.hub.docker.com
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: sctay/b-test
```

Comment:
added you to beradev, but want to push this to gcr and/or github too

Reply:
Nice! Will do.